docs/concepts/what-is-an-activity-task.md

```diff
@@ -2,9 +2,27 @@
 id: what-is-an-activity-task
 title: What is an Activity Task?
 sidebar_label: Activity Task
-description: An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+description: An Activity Task contains the context needed to make an Activity Task Execution.
 tags:
   - explanation
 ---
 
-An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+An Activity Task contains the context needed to proceed with an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution).
+From the API, users see an Activity being executed and assume that it runs once.
+
+An Activity Task is meant to run an [Activity Definition](/docs/concepts/what-is-an-activity-definition) once.
+If an Activity Task Execution fails, the Activity Task is retried in accordance with its Retry Policy.
+
+#### Context Events
+
+Activity Tasks contain Events that are needed to execute the Activity. These events are found within the Activity Task's [Context](/docs/concepts/what-is-an-activity-task#context-events).
```

Comment:
Hmm... why does this link right back to this page?
Also, an Activity Task does not contain Events - nor is it an Event.
Events live in the Workflow Execution Event History.

My current suggestion would drop the "Context Events" header and this sentence above and just have a solid set of sentences / paragraph of info.